.github/workflows/eslint.yml

```diff
@@ -0,0 +1,19 @@
+name: Lint
+
+on:
```

Comment:
File/glob filter here?

Reply:
Good point. Also, don't really need the pull_request trigger when having push. Can just have push on all branches with a path and it will work out. hmm. Unless external pull requests? Hm? How would that work out, do you know?

Reply:
Fixing it if it's an issue. Just merging. YOLO